src/main/java/com/autotune/analyzer/kruizeObject/ModelSettings.java

```diff
@@ -0,0 +1,26 @@
+package com.autotune.analyzer.kruizeObject;
+
+import java.util.List;
+
+public class ModelSettings {
+
+    private List<String> models;
+
+    public ModelSettings() {}
+
+    public List<String> getModels() {
+        return models;
+    }
+
+    public void setModels(List<String> models) {
```

Comment:
We need a validation for the models similar to what you are doing for terms. Check if the user specified model is one of `cost` or `performance`. If not fail the createExperiment call